init.lua

```diff
@@ -1 +1,4 @@
-require('starbase.main')()
+require('starbase.main')(
```

Comment:
Why is this part now requiring from custom.cfg and starbase.json? Will users need to worry about them?

Reply:
I had a lot of trouble thinking about the project settings. The solution I came up with is:

Now the main accepts two function arguments (both should be tables):
- The general base settings, things like "enable the go step" should be defined here
- The project specific settings, things like "buildtags" should go here

I also supplied two "scripts", one for requiring the `custom.cfg` module and other to load a `starbase.json` file as a table.

These are pretty hard coded defaults, but I couldn't think of a better way of doing it. What do you think?